src/components/data/EntitiesContent.js

```diff
@@ -373,7 +375,11 @@ const EntitiesContent = ({
           !snapshotName && renderDownloadButton(columnSettings),
           !_.endsWith('_set', entityKey) && renderCopyButton(entities, columnSettings),
           div({ style: { margin: '0 1.5rem', height: '100%', borderLeft: Style.standardLine } }),
-          div({ style: { marginRight: '0.5rem' } }, [`${selectedLength} row${selectedLength === 1 ? '' : 's'} selected`]),
+          div({
+            style: { marginRight: '0.5rem' },
+            role: 'status',
+            'aria-atomic': true
+          }, [`${selectedLength} row${selectedLength === 1 ? '' : 's'} selected`]),
```

Comment:
This announces the current selection status to screen readers whenever it changes. `role="status"` uses `aria-live="polite"` by default.